src/main.py

```diff
@@ -140,12 +134,11 @@ def submission_thread():
 
         print(formatted_text)
 
-        # wtf
-        for sentence in range(len(formatted_text)):
-            if len(formatted_text[sentence]) < 9:
+        for sentence in formatted_text:
+            if len(formatted_text) < 9:
                 print("Too small, skipping this element")
                 # Some gibberish value that is not present
-                formatted_text[sentence] = "999999"
+                continue
```

Comment:
Hello there!
Just want to point out something, this value of "999999" is used as a value that will not be present in the subtitles files. 
It is a bad hack and should be fixed, like making a new list with insertions. 
Probably will do it tomorrow.
Generally, I will accept all the changes except the "wtf" part. :D